worlds/terraria/Options.py

```diff
@@ -0,0 +1,39 @@
+from Options import Choice, Option
+import typing
+
+class Goal(Choice):
+    """The victory condition for your run. Stuff after the goal will not be shuffled."""
+    display_name = "Goal"
+    option_wall_of_flesh = 0
+    option_plantera = 1
+    option_moon_lord = 2
+    option_zenith = 3
+    default = 2
+
+class Achievements(Choice):
+    """
+    Adds checks upon collecting achievements. Achievements for clearing bosses and events are excluded.
+    "Exclude Grindy" also excludes fishing achievements.
+    """
+    display_name = "Achievements"
+    option_none = 0
+    option_exclude_grindy = 1
+    option_exclude_fishing = 2
+    option_all = 3
+    default = 1
+
+class FillExtraChecksWith(Choice):
+    """
+    Applies if you have achievements enabled. "Useful Items" helps to make the early game less grindy.
+    Items are rewarded to all players in your Terraria world.
+    """
+    display_name = "Fill Extra Checks With"
+    option_nothing = 0
```

Comment:
Generally speaking, if you can find some "insignificant" item to fill locations with rather than actually nothing, it's a better game experience. I would recommend creating some money items to use as filler rather than a Nothing item (1 Gold, 5 Gold, etc.).